### PR TITLE
fix search link + search firebase db name

### PR DESCRIPTION
**Pull request by choroshin**

### CODE REVIEW

File: src/components/elasticSearch/services/elasticsearch.service.js [hunk 1]
```diff
@@ -7,10 +7,14 @@
                 'ngInject';
                 var uidObj = AuthService.getAuth();
 
-                var apiPath = ENV.backendEndpoint + "/search";
+                var API_PATH = ENV.backendEndpoint + "/search";
+                var ZNK_NAME = "znk-dev";
 
+                if (!ENV.debug) {
```
Comment (assafshp) [inline src/components/elasticSearch/services/elasticsearch.service.js]: cannot use debug ?

File: src/components/adminDashboard/components/esLink/services/esLink.service.js [hunk 2]
```diff
@@ -9,8 +9,8 @@
                 var apiPath = ENV.backendEndpoint + "/invitation/assosciate_student";
                 var resetUserDataPath = ENV.backendEndpoint + "/userModule/delete";
 
-                this.createInvitationFactory = function (senderUid, senderName, receiverEmail, receiverName, senderAppName, receiverAppName, senderEmail, receiverParentEmail, receiverParentName) {
-                    return new Invitation(senderUid, senderName, receiverEmail, receiverName, senderAppName, receiverAppName, senderEmail, receiverParentEmail, receiverParentName);
+                this.createInvitationFactory = function (senderUid, receiverUid, senderName, receiverEmail, senderEmail, receiverName, senderAppName, receiverAppName) {
```
Comment (assafshp) [inline src/components/adminDashboard/components/esLink/services/esLink.service.js]: I don't understand what is the change here any way?

File: src/components/adminDashboard/components/esLink/esLink.component.js [hunk 3]
```diff
@@ -81,7 +81,7 @@
                     }
                     var student = self.selectedStudent;
                     var educator = self.selectedEducator;
-                    var invitationObj = ESLinkService.createInvitationFactory(educator.uid, student.uid, educator.name, student.email, educator.email, student.name,
+                    var invitationObj = ESLinkService.createInvitationFactory(educator.uid, student.uid, educator.nickname, student.email, educator.email, student.nickname,
```
Comment (assafshp) [inline src/components/adminDashboard/components/esLink/esLink.component.js]: we have only nickname and not name ?

Comment (choroshin) [inline src/components/adminDashboard/components/esLink/esLink.component.js]: yes

File: src/components/adminDashboard/components/esLink/services/esLink.service.js [hunk 4]
```diff
@@ -9,8 +9,8 @@
...
+                    return new Invitation(senderUid, receiverUid, senderName, receiverEmail, senderEmail, receiverName, senderAppName, receiverAppName);
                 };
 
                 this.resetUserData = function(data) {
```
Comment (choroshin) [inline src/components/adminDashboard/components/esLink/services/esLink.service.js]: remove appname

**Pull request merged**